docs/apidocs/index.rst

```diff
@@ -8,12 +8,27 @@ Qiskit Experiments API Reference
     :maxdepth: 1
 
     main
-    characterization
-    calibration
-    composite
-    randomized_benchmarking
-    tomography
-    analysis
-    data_processing
-    database_service
-   
\ No newline at end of file
+    library
+
+Utility Modules
+===============
+
+.. toctree::
+    :maxdepth: 1
+
+    mod_analysis
+    mod_calibration_management
+    mod_data_processing
+    mod_database_service
+
+Experiments Modules
+===================
+
+.. toctree::
+    :maxdepth: 1
+    
+    mod_calibration
+    mod_composite
```

Comment:
Seems like this is utility to combine experiment, rather than experiment modules.

Reply:
I moved it out of library, tbd in a future PR where it should finally live